spring-cloud-azure-samples/feature-management-web-sample/src/main/java/com/example/filters/Random.java

```diff
@@ -15,8 +15,7 @@
 
     @Override
     public boolean evaluate(FeatureFilterEvaluationContext context) {
```

Comment:
What is the context that we decide to use Random() here? And this seems to be a feature flag related change, why it is in this pr?